openwebstart/src/main/resources/i18n_de.properties

```diff
@@ -183,6 +183,7 @@ dialog.updateCheck.text=Eine neue JVM Version (Version ''{0}'' / Hersteller ''{1
 dialog.jvmManagerConfig.title=JVM Manager Konfiguration
 dialog.jvmManagerConfig.updateStrategy.text=Update Strategie:
 dialog.jvmManagerConfig.vendor.text=Hersteller:
+dialog.jvmManagerConfig.vendor.anyVendor.text=Irgendeinen Hersteller
```

Comment:
I think correct German is "Irgendein Hersteller" oder ev. "Beliebiger Hersteller" , "Ein beliebiger Hersteller"